tox.ini

```diff
@@ -4,4 +4,4 @@ skipsdist = true
 
 [testenv]
 deps = -r requirements/tests.txt
-commands = pytest
\ No newline at end of file
+commands = pytest {posargs}
```

Comment:
What's `posargs`?

Reply:
the way to pass tox CLI arguments to pytest
https://tox.readthedocs.io/en/latest/example/general.html#interactively-passing-positional-arguments